src/presenter/board-presenter.js

```diff
@@ -14,19 +16,21 @@ export default class BoardPresenter {
   #eventListComponent = new EventsList();
   #noPointComponent = new NoPoint();
   #filterElement = new FilterSection();
-
+  #currentSortParameter = SortParameters.DATE_FROM;
   #boardPoints = [];
+  #srcBoardPoints = [];
   #pointPresenter = new Map();
 
 
   constructor(boardContainer, filterContainer, boardPoints) {
-    this.#boardPoints = [...boardPoints];
+    this.#boardPoints = [...boardPoints].sort((a, b) => sortMethod.SORT_DAY(a, b));
```

Comment:
зачем создаешь новую функцию, если можно просто передать ссылку на имеющуюся функцию, с такой же сигнатурой?
```suggestion
    this.#boardPoints = [...boardPoints].sort(sortMethod.SORT_DAY);
```